frontends/infinite-corridor/src/pages/HomePage.tsx

```diff
@@ -141,6 +227,70 @@ const HomePage: React.FC = () => {
           </div>
         </Grid>
       </GridContainer>
+      <TabContext value={tabValue}>
+        <GridContainer>
+          <GridColumn variant="single-full">
+            <h3>Upcoming Courses</h3>
+            <TabList className="courses-nav" onChange={handleChange}>
+              <Tab label="All" value="all" />
+              <Tab label="Certificate" value="certificate" />
+              <Tab label="Micromasters" value="micromasters" />
+              <Tab label="Professional ed" value="professional" />
+            </TabList>
+            <TabPanel value="all" className="courses-carusel">
```

Comment:
**Request:** `className="courses-carusel"` on all or none of the tabpanels.

(The spacing jumps when you switch tabs.)